app/views/namespaces/index.html.slim

```diff
@@ -1,5 +1,10 @@
 = render 'namespaces/partials/special_namespaces'
 
-= render 'namespaces/partials/accessible_namespaces'
+= render 'namespaces/partials/normal_namespaces'
 
 = render 'namespaces/partials/components_templates'
+
+/ TODO: move this to be used as a component prop instead of global value
```

Comment:
Can you do this for this PR ?

Reply:
My idea was to refactor this later because I'll change a little bit the approach in the page file. The problem is that the page file is not a component itself, being just something that bootstraps the vue components. 

The info below is used only in this file. We could attach this info to some HTML element (not the properly component I'm talking about) to hide it from the global scope but in the end it's the same thing.

Reply:
Sounds good.